packages/ui/src/containers/TermsOfUse/TermsOfUseConfirmModal/index.test.tsx

```diff
@@ -1,20 +1,16 @@
-import { render } from '@testing-library/react';
 import React from 'react';
 
+import renderWithPageContext from '@/__mocks__/RenderWithPageContext';
```

Comment:
well the file name and its usage are confusing. is it a React component?

Reply:
<img width="1375" alt="image" src="https://user-images.githubusercontent.com/36393111/176065647-73177616-bc33-4a82-97a1-0399dd8a07af.png">

it's a render method, wrapup react-library's render method with a ContextProvide.  The Lint rule requires the filename to be like that. Since it's mock, just keep it that way.